vector/src/main/res/layout/item_suggested_room.xml

```diff
@@ -102,6 +102,17 @@
         app:barrierDirection="bottom"
         app:constraint_referenced_ids="roomAvatarBottomSpace" />
 
+    <TextView
+        android:id="@+id/inlineErrorText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:drawablePadding="8dp"
+        android:padding="8dp"
+        android:textColor="@color/design_default_color_error"
```

Comment:
Please use color from the theme: `?colorError`

Reply:
done